src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -200,7 +198,8 @@ def download_logs(
                     # Skip sleeping on last retry
                     if i < MAX_RETRIES - 1:
                         log.info(f"Going to retry in {retry_interval} seconds")
-                        time.sleep(retry_interval)
+
+                    [time.sleep(1) for _ in range(retry_interval)]  # allow Ctrl+C when running locally
```

Comment:
when not using value of the list-comprehension, I prefer:
```suggestion
                    # allow Ctrl+C when running locally
                    for _ in range(retry_interval):
                        time.sleep(1)
```